api/src/main/java/jakarta/jws/soap/SOAPMessageHandler.java

```diff
@@ -35,7 +35,7 @@
     /**
      * List of SOAP headers processed by the handler.  Each element in this array contains a QName which defines the
      * header element processed by the handler.  The QNames are specified using the string notation described in the
-     * documentation for javax.xml.namespace.QName.valueOf(String qNameAsString)
+     * documentation for jakarta.xml.namespace.QName.valueOf(String qNameAsString)
```

Comment:
this is not to be changed (part of JAXP/java.xml module on jdk 9+)

Reply:
It should be fine now